test/dispatchers/dispatcherTests.ts

```diff
@@ -32,7 +32,7 @@ describe("Dispatchers", () => {
       var callbackWasCalled = false;
       (<any> dispatcher)._eventToCallback["click"] = () => callbackWasCalled = true;
 
-      var callback = () => null;
+      var callback = () => { return; };
```

Comment:
I'm running into an issue with the compiler where leaving it as is has the compiler complaining that there is an implicit `any`.

I could explicitly say that this function returns an `any`, but after thinking about it, since the point of this callback is to do nothing, I decided to change it to just do nothing.